src/ansible_navigator/actions/run.py

```diff
@@ -418,7 +420,7 @@ def _init_replay(self) -> bool:
             return False
 
         version = data.get("version", "")
-        if version.startswith("1."):
+        if version.startswith("1.") or version.startswith("2."):
```

Comment:
Looks like this condition might need to be updated on every major release. Can't we use `>=` operator.

Reply:
This was an odd one, because the addition of settings wasn't a breaking change for the TUI, but I updated the version of the artifact to match the major release coming out.

What we really need is a json schema for the artifact file that checks for at least the keys we care about, but there isn't really time for that right now.

I'll add an issue about creating a schema file for it, then we can just check it's sanity in the future.  There's no guarantee it hasn't been modified since navigator wrote it so it's probably a good check to add.

Reply:
https://github.com/ansible/ansible-navigator/issues/1159